src/main/java/io/reactivex/Observable.java

```diff
@@ -11344,6 +11328,7 @@ public final void subscribe(Observer<? super T> observer) {
      *         within time windows defined by the timeout selectors
      * @throws NullPointerException
      *             if {@code itemTimeoutIndicator} is null
+     *             if {@code other} is null
```

Comment:
You might want to look at the rendered Javadoc here. I'm betting these get smashed together with only a single space. Might be better to switch to `{@code itemTimeoutIndicator} or {@code other} is null`.